openapiart/common.py

```diff
@@ -625,6 +670,12 @@ def append(self, item):
     def clear(self):
         del self._items[:]
         self._index = -1
+    
+    def set(self, index, item):
+        if isinstance(item, OpenApiObject) is False:
```

Comment:
Validate if item is really that iterobj. Otherwise it use will do this
```python
pc = api.prefix_config()
pc.deserialize(payload)
f = pc.f
j = pc.j
j.set(0, f)    # add validation
```

Reply:
Agreed, I had the same concern while implementing. But for "append" call too the logic is same, so I followed it. will add the validation